src/source/Common/Lws/LwsCall.c

```diff
@@ -73,6 +73,10 @@ STATUS blockingLwsCall(PRequestInfo pRequestInfo, PCallInfo pCallInfo)
 
 CleanUp:
 
+    if(retStatus == STATUS_IOT_CREATE_LWS_CONEXT_FAILED)
+    {
+        DLOGW("Unable to create context or connect to websocket server");
```

Comment:
/nit can we make the warn msg ""Unable to create LWS context or connect to Websocket server"